Attached data disk encryption

angarg05 opened the pull request:
Added text for Attached data disk encryption, as per SQL Server team and Saket for the upcoming Ignite (2022) announcement

angarg05 committed:
Attached data disk encryption

Added text for Attached data disk encryption, as per SQL Server team and Saket for the upcoming Ignite (2022) announcement

angarg05 committed:
Update confidential-vm-overview.md

removed en-us from links

RunCai commented on articles/confidential-computing/confidential-vm-overview.md:
```diff
@@ -42,12 +42,18 @@ Confidential OS disk encryption is optional, because this process can lengthen t
 From July 2022, encrypted OS disks will incur higher costs. This change is because encrypted OS disks use more space, and compression isn't possible. For more information, see [the pricing guide for managed disks](https://azure.microsoft.com/pricing/details/managed-disks/).
 
+## Attached data disk encryption
+If you are using an Azure confidential VM and attaching extra disks, in order to maximize its security benefits follow the below recommendation:
```
below recommendation has lots of implications to customer.
Please document it explicitly.

RunCai reviewed (commented):
Please document the implications of extending the BitLocker to data disk, another question is BitLocker only works for Windows VM, what is the recommendation for Linux VM?

angarg05 committed:
Update confidential-vm-overview.md

updated timestamp

angarg05 committed:
Update confidential-vm-overview.md

Adding condition for Windows vs Linux VMs, per Run's recommendation

saketmicrosoft committed:
Adding relative links instead of absolute links

We need to review if these links work

RunCai reviewed (changes requested):
please do not merge the this PR to main doc. @angarg05 , Vikas and I think we should not merge the change into CVM overview page, and instead we should update in CVM FAQ page with additional manual steps documented and reviewed by engineering team.

saketmicrosoft committed:
Updating the data disk question

angarg05 committed:
Merge pull request #2 from saketmicrosoft/patch-4

Updating the data disk question

angarg05 committed:
Merge branch 'MicrosoftDocs:main' into patch-2

angarg05 committed:
Merge pull request #1 from saketmicrosoft/patch-3

Adding relative links instead of absolute links

angarg05 committed:
Update confidential-vm-overview.md

deleted changes from overview, added information about disk encryption to CVM FAQ
